lib/puppet_x/puppetlabs/scheduled_task/taskscheduler2_v1task.rb

```diff
@@ -0,0 +1,279 @@
+# This class is used to manage V1 compatible tasks using the Task Scheduler V2 API
```

Comment:
Nit - this is not a Facade, it's an Adapter.

Reply:
I'll reword.  Aren't metaphors wonderful!  People can have different interpretations.

Reply:
Reworded.